pom.xml

```
@@ -5,15 +5,28 @@
 	<parent>
 		<groupId>org.jenkins-ci.plugins</groupId>
 		<artifactId>plugin</artifactId>
-		<version>1.541</version>
+		<version>4.29</version>
 	</parent>
 
 	<artifactId>hidden-parameter</artifactId>
 	<packaging>hpi</packaging>	
-	<version>0.0.5-SNAPSHOT</version>
+	<version>${revision}${changelist}</version>
 	<name>Hidden Parameter plugin</name>
     <url>http://wiki.jenkins-ci.org/display/JENKINS/Hidden+Parameter+Plugin</url>
```

This looks great @MadsJakobsen !  Thanks so much for doing it.

If you're willing to make one more commit, you can convert the plugin to use documentation from the GitHub repository README file rather than using the unmaintained copy taken from the wiki.

See my proposed pull request from last year, #4 to fix the formatting in the README file.
```suggestion
    <url>https://github.com/jenkinsci/hidden-parameter-plugin</url>
```

---

absolutely 👍